package.json

```diff
@@ -20,6 +20,6 @@
     "frontend-build:development" : "./node_modules/gulp/bin/gulp.js build:development",
     "frontend-build:production" : "./node_modules/gulp/bin/gulp.js build:production",
     "frontend-build:watch" : "./node_modules/gulp/bin/gulp.js watch",
-    "postinstall": "npm run frontend-install"
+    "postinstall": "npm run frontend-install && npm run frontend-build:production"
```

Comment:
Agree. Any changes to these scripts or the gulpfile should be the same across all apps, otherwise it gets a bit painful.